### PR TITLE
Expose storageClass param

### PR DESCRIPTION
**Pull request by liyanwei93**
Expose storageClass param for cp4waiops install. @morningspace

### CODE REVIEW

File: config/cp4waiops/3.3/values.yaml [hunk 1]
```diff
@@ -6,3 +6,5 @@ cp4waiops32:
       namespace: cp4waiops
     size: small
     dockerPassword: REPLACE_IT
+    storageClass: rook-cephfs
+    storageClassLargeBlock: rook-cephfs
```
Comment (morningspace) [inline config/cp4waiops/3.3/values.yaml]: Missing newline at the end.

File: docs/gitops-deploy-cp4waiops-demo.md [hunk 2]
```diff
@@ -90,6 +90,8 @@ Below table summarizes the detailed meaning for each parameter:
 | argocd.allowLocalDeploy   | bool   | true               | Allow applications to be deployed on the same cluster where Argo CD runs.
 | cp4waiops.enabled         | bool   | true               | Specify whether or not to install CP4WAIOps.
 | cp4waiops.instanceName    | string | aiops-installation | The instance name of CP4WAIOps.
+| cp4waiops.storageClass         | string   | rook-cephfs | The storage class that you want to use
+| cp4waiops.storageClassLargeBlock | string | rook-cephfs | The storage class Large Blockt hat you want to use.
```
Comment (morningspace) [inline docs/gitops-deploy-cp4waiops-demo.md]:
```suggestion
| cp4waiops.storageClassLargeBlock | string | rook-cephfs | The storage class large block  that you want to use.
```

File: config/cp4waiops/3.3/values.yaml (hunk 1 above)
Comment (liyanwei93) [reply config/cp4waiops/3.3/values.yaml]: Fixed

File: docs/gitops-deploy-cp4waiops-demo.md (hunk 2 above)
Comment (liyanwei93) [reply docs/gitops-deploy-cp4waiops-demo.md]: Fixed

**Pull request merged**